PocketKit/Tests/PocketKitTests/ArchivedItemsViewModelTests.swift

```diff
@@ -79,7 +79,11 @@ class ArchivedItemsViewModelTests: XCTestCase {
     }
 
     func test_deleteAction_delegatesToSource_andUpdatesSnapshot() {
-        source.stubDelete { _ in }
+        let expectDeleteCall = expectation(description: "expect a source.delete(archivedItem:)")
+        source.stubDelete { _ in
+            expectDeleteCall.fulfill()
+        }
+
```

Comment:
Love it. Back to the pre-predicate speeds we go!

Reply:
😎